website_sale_preset_carts_signup/controllers/auth_signup.py

```diff
@@ -31,4 +32,3 @@ def web_auth_signup(self, *args, **kw):
                    .get_subscriptions()
         )
         return res
-

```

Comment:
This EOL should not be removed I think

Reply:
that's strange because I have a EOL locally...

![image](https://user-images.githubusercontent.com/7766650/95427800-6adb8a00-0948-11eb-9eb7-22e361ec450c.png)

Reply:
👍